### DISCUSSION TITLE
Stable version of ksh93

### DISCUSSION THREAD

**Discussion by asureshkumars** [ksh93/ksh#787] [Q&A] [answered]
Hi,
Im currently using ksh93 that was ported during 2012.
# ksh93 --version
  version         sh (AT&T Research) 93u+ 2012-08-01

Can you please let me know which is a stable version under 1.0.x version that can be used. ?
Can you please help if there are good test coverage for this new features added in these versions ?

**Comment by posguy99** [comment, marked as answer]
The current release version is 1.0.10, available from https://github.com/ksh93/ksh/releases

As for testing... yes, u+m has extensive regression testing.
